config/routes.rb

```diff
@@ -54,8 +54,6 @@
       end
 
       resource :claim_details, only: [:show]
-      resource :review_and_adjusts, only: [:show]
```

Comment:
Was this a mistake? Disagrees with https://github.com/ministryofjustice/nsm-e2e-test/pull/112

Reply:
so changed it twice so will need a new e2e PR... :)